### ISSUE TITLE
Unable to start stream with youtube app on linux

### ISSUE THREAD

**Issue by DevBennyturtle** [openairplay/airplay2-receiver#8]
I was unable to get this working on linux by installing requirements.txt and using the wifi interface wlp1s0
If there is anything I can do to test or to send more info (please say what) let me know 

Also I meant to ask this but do you guys have a way to communicate like discord? I wanted to contribute to this project but dont know where to start
```ALSA lib pcm.c:2642:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.rear
ALSA lib pcm.c:2642:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.center_lfe
ALSA lib pcm.c:2642:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.side
ALSA lib pcm_route.c:869:(find_matching_chmap) Found no matching channel map
ALSA lib pcm_route.c:869:(find_matching_chmap) Found no matching channel map
ALSA lib pcm_route.c:869:(find_matching_chmap) Found no matching channel map
ALSA lib pcm_route.c:869:(find_matching_chmap) Found no matching channel map
Cannot connect to server socket err = No such file or directory
Cannot connect to server request channel
jack server is not running or cannot be started
JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for -1, skipping unlock
JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for -1, skipping unlock
Cannot connect to server socket err = No such file or directory
Cannot connect to server request channel
jack server is not running or cannot be started
JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for -1, skipping unlock
JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for -1, skipping unlock
ALSA lib pcm_oss.c:377:(_snd_pcm_oss_open) Unknown field port
ALSA lib pcm_oss.c:377:(_snd_pcm_oss_open) Unknown field port
ALSA lib pcm_usb_stream.c:486:(_snd_pcm_usb_stream_open) Invalid type for card
ALSA lib pcm_usb_stream.c:486:(_snd_pcm_usb_stream_open) Invalid type for card
Cannot connect to server socket err = No such file or directory
Cannot connect to server request channel
jack server is not running or cannot be started
JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for -1, skipping unlock
JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for -1, skipping unlock
TEARDOWN rtsp://192.168.1.16/2078644956552382407
Content-Length: 84
Content-Type: application/x-apple-binary-plist
CSeq: 13
DACP-ID: BB392EBCC141E777
Active-Remote: 1447734394
User-Agent: AirPlay/521.1.3```

**Comment by LewdNeko** [comment]
Could you test with the latest changes and confirm if this is still an issue?

**Comment by systemcrash** [comment]
Please, can you test with latest master, and also try PortAudio 19.7? Lots of fixes between .6 and .7

**Comment by DevBennyturtle** [comment]
I will do that now, thank you for letting me know

**Comment by DevBennyturtle** [comment]
I can no longer see "myap2" on my iphone, and i tried running it with python2 and it no longer works as pyav doesnt support python2 anymore

**Comment by systemcrash** [comment]
Not sure what to say. Might depend on your Linux distro. Python2 is unsupported already for 18 months.

**Comment by LewdNeko** [comment]
Possibly fixed by https://github.com/openairplay/airplay2-receiver/commit/55833b53210116831abeebd0a63dacbbe6949b45 ?

**Comment by systemcrash** [comment]
YouTube and other media apps behave a little strangely via airplay, but on the whole, they can successfully send sound to this receiver. Loading successive tracks is where it sometimes disconnects, but that seems unrelated to this. Closing. 

> Possibly fixed by [55833b5](https://github.com/openairplay/airplay2-receiver/commit/55833b53210116831abeebd0a63dacbbe6949b45) ?

Agreed.
